### ISSUE TITLE
Handle duplicate connection strings in configuration file

### ISSUE THREAD

**Issue by prestonvasquez** [alpstable/gidari#163]
A user could create a configuration file with duplicate connection strings like this:

```yml
url: https://official-joke-api.appspot.com
connectionStrings: 
  - mongodb://mongo1:27017/jokes
  - mongodb://mongo1:27017/jokes
rateLimit:
  burst: 5
  period: 1
requests:
  - endpoint: /random_joke
```

If this happens, we need to log a warning in `transport.Config#repos` that the user did this, and only create a repo for one of the connection strings.

**Comment by krishnaindani** [comment]
hey, could I take a stab at this? Would like to work if open.

**Comment by prestonvasquez** [comment]
@krishnaindani You're assigned!

**Comment by krishnaindani** [comment]
Thanks

**Comment by krishnaindani** [comment]
The way I was thinking is to use a map[string]bool while we loop to make newTX here https://github.com/alpine-hodler/gidari/blob/main/internal/transport/transport.go#L269
 and maintaining a dns entry in map that had we have already looped over. And if duplicated just log it. How does this sounds or would you be able to recommend any better idea here?

**Comment by prestonvasquez** [comment]
@krishnaindani That sounds reasonable to me! In order to maintain the complexity requirements for Hacktoberfest, do you think you could take a look at Issue #180 and combine the two into one PR?

**Comment by krishnaindani** [comment]
Sure will take a look

**Comment by krishnaindani** [comment]
Could you assign me that issue?

**Comment by prestonvasquez** [comment]
@krishnaindani Thank you! Since you have not made your first contribution can you comment on that ticket so you are in the scope of assignable engineers?

**Comment by prestonvasquez** [comment]
@krishnaindani Are you still interested in pursuing this issue?

**Comment by prestonvasquez** [comment]
Closed by #360
